telosnetwork/teloscan
# #219 | Brave mobile metamask issue

# Fixes #219 

## Description
This PR fixes an issue where, if you are on Brave browser on mobile, attempting to launch the metamask app will instead launch Brave Wallet (which is not presently supported by Teloscan)

## Test scenarios
- on a mobile device, download brave browser
- go to the dev preview at https://deploy-preview-265--testnet-teloscan.netlify.app/
- click the `CONNECT WALLET` button in the top right corner
    - in the login modal, the text should say "Continue on MetaMask"
    - tapping the MM logo should open MetaMask app / direct you to install MM
- repeat the same steps on a desktop browser with metamask installed
    - the login modal textshould say "metamask"
    - clicking the MM logo should open a metamask window as usual

## Checklist:
